pkg/api/dtos/annotations.go

```diff
@@ -13,19 +13,21 @@ type PostAnnotationsCmd struct {
 }
 
 type UpdateAnnotationsCmd struct {
-	Id      int64    `json:"id"`
-	Time    int64    `json:"time"`
-	TimeEnd int64    `json:"timeEnd,omitempty"` // Optional
-	Text    string   `json:"text"`
-	Tags    []string `json:"tags"`
+	Id      int64            `json:"id"`
+	Time    int64            `json:"time"`
+	TimeEnd int64            `json:"timeEnd,omitempty"` // Optional
+	Text    string           `json:"text"`
+	Tags    []string         `json:"tags"`
+	Data    *simplejson.Json `json:"data"`
```

Comment:
Maybe this could be an interface{} instead so it is more straight forward to unmarshal into types.

Reply:
@kylebrandt I don't really understand the benefit? Could you provide an example? The corresponding storage type field has the same type at least.